client/server/lib/is-mobile-app/index.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Returns whether source matches a WordPress mobile app.
+ * @param source string
+ * @returns {boolean} Whether source matches the ones used on the WordPress mobile apps.
+ */
+export function isWpMobileApp( source ) {
+	return /wp-(android|iphone)/.test( source );
+}
+
+/**
+ * Returns whether user is using a WooCommerce mobile app.
+ * @param source string
+ * @returns {boolean} Whether source matches the ones used on the WooCommerce mobile apps.
+ */
+export function isWcMobileApp( source ) {
```

Comment:
Can we update this to `isWooMobileApp` I think this will would make it more clear.

Reply:
I followed the naming we already have in Calypso for the WooCommerce app check. 
You can see it in this package: https://github.com/Automattic/wp-calypso/tree/trunk/client/lib/mobile-app

I feel that if we change it here, we will need to change it in the client too, and multiple files are already using it; The refactor will take a good number of files. 

Let me know what you think and if you want to apply that refactor 💪

Reply:
Thanks for the clarification. @sbarbosa ! 
I thought it was something new we are adding. 

Lets keep things as they are, isWooMobileApp still reads (IMHO) better but we can do that change down the line at a later time if at all. :)